examples/bzlmod/entry_point/test_entry_point.py

```diff
@@ -38,6 +39,58 @@ def test_entry_point(self):
         )
         self.assertEqual(proc.stdout.decode("utf-8").strip(), "yamllint 1.28.0")
 
+    def test_pylint_entry_point(self):
+        rlocation_path = os.environ.get("PYLINT_ENTRY_POINT")
+        assert (
+            rlocation_path is not None
+        ), "expected 'PYLINT_ENTRY_POINT' env variable to be set to rlocation of the tool"
+
+        entry_point = pathlib.Path(runfiles.Create().Rlocation(rlocation_path))
+        self.assertTrue(entry_point.exists(), f"'{entry_point}' does not exist")
+
+        proc = subprocess.run(
+            [str(entry_point), "--version"],
+            check=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        self.assertRegex(proc.stdout.decode("utf-8").strip(), "^pylint 2\.15\.9")
+
+    def test_pylint_entry_point_deps(self):
+        rlocation_path = os.environ.get("PYLINT_ENTRY_POINT")
+        entry_point = pathlib.Path(runfiles.Create().Rlocation(rlocation_path))
```

Comment:
Should we run `self.assertTrue(entry_point.exists(), f"'{entry_point}' does not exist")`